src/unix/fs.c

```diff
@@ -217,6 +217,30 @@ static ssize_t uv__fs_mkdtemp(uv_fs_t* req) {
 }
 
 
+static int uv__fs_mkstemp(uv_fs_t* req) {
+  static int no_cloexec_support;
+  int fd;
+
+#ifdef O_CLOEXEC
+  no_cloexec_support = 1;
```

Comment:
In this case we're using FD_CLOEXEC not O_CLOEXEC (which is passed to open(2)). AFAIK that is supported on all Unix platforms.